recipes/katago/LICENSE

```diff
@@ -0,0 +1,35 @@
+The code in this repository currently relies on several other libraries, parts of libraries,
```

Comment:
I don't think this file is needed. If it's in the source tarball it'll be taken from there (the key in the `meta.yaml` is still needed).